contracts/pcv/snapshot/SnapshotDelegatorPCVDeposit.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "../PCVDeposit.sol";
+
+interface DelegateRegistry {
+    function setDelegate(bytes32 id, address delegate) external;
+
+    function clearDelegate(bytes32 id) external;
+
+    function delegation(address delegator, bytes32 id) external view returns(address delegatee);
+}
+
+/// @title Snapshot Delegator PCV Deposit
+/// @author Fei Protocol
+contract SnapshotDelegatorPCVDeposit is PCVDeposit {
+
+    event DelegateUpdate(address indexed oldDelegate, address indexed newDelegate);
+
+    /// @notice the Gnosis delegate registry used by snapshot
+    DelegateRegistry public constant DELEGATE_REGISTRY = DelegateRegistry(0x469788fE6E9E9681C6ebF3bF78e7Fd26Fc015446);
+    
+    /// @notice the token that is being used for snapshot
+    IERC20 public immutable token;
+
+    /// @notice the keccak encoded spaceId of the snapshot space
+    bytes32 public spaceId;
+    
+    /// @notice the snapshot delegate for the deposit
+    address public delegate;
+
+    /// @notice Snapshot Delegator PCV Deposit constructor
+    /// @param _core Fei Core for reference
+    /// @param _token snapshot token
+    /// @param _spaceId the id (or ENS name) of the snapshot space
+    constructor(
+        address _core,
+        IERC20 _token,
+        bytes32 _spaceId,
+        address _initialDelegate
+    ) CoreRef(_core) {
+        token = _token;
+        spaceId = _spaceId;
+        _delegate(_initialDelegate);
+    }
+
+    /// @notice withdraw tokens from the PCV allocation
+    /// @param amountUnderlying of tokens withdrawn
+    /// @param to the address to send PCV to
+    function withdraw(address to, uint256 amountUnderlying)
```

Comment:
I think it's awkward to have a function that behaves exactly the same way as `PCVDeposit::withdrawERC20` but with just more relaxed pre-conditions.

Reply:
Removed the pause condition